livre/eq/Window.cpp

```diff
@@ -69,10 +69,12 @@ struct Window::Impl
                         pipe->getFrameData()->getVRParameters().getMaxGPUCacheMemoryMB();
         _textureCache.reset( new TextureCache( node->getTextureDataCache(),
                                                maxGpuMemory * LB_1MB, GL_LUMINANCE8 ));
-
-        const size_t computeThreads = 4;
+        const size_t renderThreads = 2;
+        const size_t computeThreads = 2;
         const size_t uploadThreads = 4;
```

Comment:
Personally I don't like env variables. They are obscured and you have to go to the documentation. "--help" I like better but it is just a preference, not hard boilled feelings about it.